src/index.ts

```diff
@@ -89,56 +89,60 @@ const server = createServer(async (request, response) => {
 
   console.time("tool-call");
   const toolCaller = await capiClient.chat.completions.create({
+    // @ts-expect-error - type error due to Copilot/OpenAI SDKs interop, I'll look into it ~@gr2m
```

Comment:
I'd opt to wait until we've resolved this to ship

Reply:
it has been removed via #8